import_tracker/web_client/main.js

```diff
@@ -40,6 +41,30 @@ wrap(S3ImportView, 'render', function (render) {
 
     this.$('.form-group').last().after(excludeExistingInput);
 });
+
+const setBrowserRoot = (browserWidget, destId) => {
+    const folderId = destId.trim().split(/\s/)[0];
```

Comment:
The destination could be a folder, user, or collection.  This is stored in the destinationType.

Reply:
This should be covered now. The browser attempts to use the `Destination type` selected model, if that fails to find a corresponding model with the `destId`, it shows an error dialog. Since the `Destination type` field is automatically set after resource selection the details are relatively abstracted and it seems intuitive to use